tests/integration/src/test/java/org/apache/pulsar/tests/integration/suites/PulsarTieredStorageTestSuite.java

```diff
@@ -58,8 +58,8 @@ public void setupCluster() throws Exception {
             brokerContainer.withEnv("managedLedgerMaxEntriesPerLedger", String.valueOf(ENTRIES_PER_LEDGER));
             brokerContainer.withEnv("managedLedgerMinLedgerRolloverTimeMinutes", "0");
             brokerContainer.withEnv("managedLedgerOffloadDriver", "s3");
-            brokerContainer.withEnv("bucket", "pulsar-integtest");
-            brokerContainer.withEnv("endpoint", "http://" + S3Container.NAME + ":9090");
+            brokerContainer.withEnv("s3ManagedLedgerOffloadBucket", "pulsar-integtest");
```

Comment:
It would be good to test both the BC version and the variants. 
Maybe ```
if (i++ % 2) {
    // use BC variant
} else {
    // use new variant
}
```

Kinda messy though.

Reply:
This brings up two questions:

1) Should I just create a second class for the BC tests, e.g. PulsarTieredStorageBCTestSuite?
2) How are the following properties being propagated to the JCloud provider?  I didn't see any place in the code (old or new) that does this, which has me concerned that I might have missed something.

 brokerContainer.withEnv("managedLedgerMaxEntriesPerLedger", String.valueOf(ENTRIES_PER_LEDGER));
 brokerContainer.withEnv("managedLedgerMinLedgerRolloverTimeMinutes", "0");

Reply:
PulsarTieredStorageTestSuite is a base class used by TestS3Offload. So adding a new TestSuite class would be kinda messy. I'm happy enough to keep it as it is now. When you add azure, I assume there'll be some tests that use the new values.

These values get passed as environment values to the docker containers which run the tests. The docker containers go through each environment variable, sees if it matches a field in the configuration file (like broker.conf), and if so updates that field.